README.zh-CN.md

```diff
@@ -12,38 +12,103 @@
 </div>
 
 ----
-
-<a href="https://ava.antv.antgroup.com"><img src="https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*rXVYRJ0EMDsAAAAAAAAAAAAADmJ7AQ/original" align="left"  width="180"  hspace="10" vspace="6"></a>
-
-**AVA** (<img src="https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*QzIsSrfsCW0AAAAAAAAAAAAADmJ7AQ/original" width="16"> Visual Analytics) 是为了更简便的可视分析而生的技术框架。 其名称中的第一个 **A** 具有多重涵义：一个智能驱动（*AI driven*）、自动化（*Automated*）、支持增强分析（*Augmented*）的可视分析解决方案。
-
+## ❓ 什么是AVA
```

Comment:
What is your consideration about adding emojis before the title?

Reply:
I referenced G2. https://github.com/antvis/G2/blob/v5/README.md

Reply:
but why

Reply:
From the perspective of rigor, the emoji should be removed. done～